plugins.d/alarm-notify.sh

```diff
@@ -220,7 +220,7 @@ fi
 images_base_url="https://registry.my-netdata.io"
 
 # curl options to use
-curl_options="--insecure"
+#curl_options=""
```

Comment:
Almost, but not quite.  This still needs to not be commented here (yes, it's a bit confusing, but shell scripting wasn't exactly designed for stuff this complex).